WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -15,7 +15,7 @@ msgstr ""
 
 #. translators: Subtitle to be displayed below the application name in the Apple App Store. Limit to 30 characters including spaces and commas!
 msgctxt "app_store_subtitle"
-msgid "Make your WordPress site faster and safer."
+msgid "Faster, safer WordPress"
```

Comment:
As @oguzkocer raised when he covered for me releasing version 18.5, the Jetpack subtitle has been longer that 30 characters all this time.

We've discussed it internally and this is the new, shorter version at 23 characters. Hopefully all localizations will keep it below the limit 🤞

Reply:
![image](https://user-images.githubusercontent.com/216089/142190825-4c99bd5a-e210-483f-b662-d072647aab0e.png)